apps/bridge-app/package.json

```diff
@@ -17,18 +17,9 @@
     "@eth-optimism/op-app": "*",
     "@eth-optimism/tokenlist": "^9.0.12",
     "@eth-optimism/ui-components": "*",
-    "@radix-ui/react-avatar": "^1.0.4",
-    "@radix-ui/react-dialog": "^1.0.5",
-    "@radix-ui/react-dropdown-menu": "^2.0.6",
-    "@radix-ui/react-label": "^2.0.2",
-    "@radix-ui/react-separator": "^1.0.3",
-    "@radix-ui/react-slot": "^1.0.2",
-    "@radix-ui/react-switch": "^1.0.3",
-    "@radix-ui/react-tabs": "^1.0.4",
     "@rainbow-me/rainbowkit": "2.0.0-beta.1",
     "@remixicon/react": "^4.1.1",
     "@tanstack/react-query": "^5.10.0",
-    "class-variance-authority": "^0.7.0",
```

Comment:
not being used in the `bridge-app` - mostly a utility used when creating the reusable components